code/datums/gamemode/role/ninja.dm

```diff
@@ -523,6 +553,68 @@ Helpers For Both Variants
 	else
 		return ..()
 
+		
+/*=======
+Suit and assorted
+=======*/
+
+/obj/item/clothing/head/helmet/space/ninja
+	name = "ninja hood"
+	desc = "What may appear to be a simple black garment is in fact a highly sophisticated nano-weave helmet. Standard issue ninja gear."
+	icon_state = "s-ninja"
+	item_state = "s-ninja"
+	armor = list(melee = 60, bullet = 50, laser = 30,energy = 15, bomb = 30, bio = 30, rad = 25)
+	species_fit = list("Human")
+	species_restricted = list("Human")
+	eyeprot = 0
+	body_parts_covered = FULL_HEAD|BEARD
+
+/obj/item/clothing/suit/space/ninja
+	name = "ninja suit"
+	desc = "A unique, vacuum-proof suit of nano-enhanced armor designed specifically for Spider Clan assassins."
+	icon_state = "s-ninja"
+	item_state = "s-ninja_suit"
+	slowdown = NO_SLOWDOWN
+	body_parts_covered = ARMS|LEGS|FULL_TORSO
+	armor = list(melee = 60, bullet = 50, laser = 30,energy = 15, bomb = 30, bio = 30, rad = 30)
```

Comment:
Fairly beefy armor, 75% chance of a bullets damage being halved, 25% chance of it doing no damage at all, and all that combined with a shield weapon.

Reply:
Lowered it to security gear levels.